e2e/namespace/install/cli/run_test.go

```diff
@@ -36,78 +36,75 @@ import (
 	v1 "github.com/apache/camel-k/pkg/apis/camel/v1"
 )
 
-func TestRunExamplesFromGitHub(t *testing.T) {
+func TestRun(t *testing.T) {
```

Comment:
Given these names show up the test log and can be a great way of finding the "needle in the haystack", can you make this less generic?